<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProduksi</class>
 <widget class="QWidget" name="FormProduksi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDProduksiLabel">
      <property name="text">
       <string>ID Produksi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iDProduksiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tanggalProduksiLabel">
      <property name="text">
       <string>Tanggal Produksi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="tanggalProduksiDateEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jumlahUtuhLabel">
      <property name="text">
       <string>Jumlah Utuh</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="jumlahUtuhLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jumlahRetakLabel">
      <property name="text">
       <string>Jumlah Retak</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="jumlahRetakLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="totalLabel">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="totalLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="iDPopulasiLabel">
      <property name="text">
       <string>ID Populasi</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="iDPopulasiComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
